lib/resque/scheduler.rb

```diff
@@ -276,6 +276,8 @@ def enqueue_from_config(job_config)
             # CustomJobClass). Otherwise, pass off to Resque.
             if klass.respond_to?(:scheduled)
               klass.scheduled(queue, klass_name, *params)
+            elsif klass.respond_to?(:perform_later)
+              klass.set(queue: queue).perform_later(*params)
```

Comment:
Could you add a one-line comment here about what is code is doing? Even just linking to this PR would be helpful for the future reader.